### ISSUE TITLE
Allow LaTeX style dashes

### ISSUE THREAD

**Issue by jnferner** [squid-lang/squid#30]
`-` is hyphen  
`--` is en-dash  
`---` is em-dash

**Comment by bash** [comment]
I propose that we automatically remove spaces around en- and em-dashes.

**Comment by jnferner** [comment]
Agreed, everything else is just simply wrong. The user can still go around this restriction if they wish by using pre-formatted blocks.

**Comment by jnferner** [comment]
Additional idea: If a hyphen is surrounded by spaces, format it as a minus: −
